src/lib/evil/evil_stdlib.h

```diff
@@ -73,7 +73,11 @@ EAPI int unsetenv(const char *name);
  *
  * @since 1.8.0
  */
-EAPI char *mkdtemp(char *__template);
+UNIMPLEMENTED inline EAPI char *mkdtemp(char *__template)
```

Comment:
Maybe is a good idea to add this and those other UNIMPLEMENTED functions to an Issue?

Reply:
Yep, it is.